articles/azure-large-instances/workloads/epic/faq.md

```diff
@@ -0,0 +1,91 @@
+---
+title: FAQ
+description: Provides resolutions for common issues that arise in working with ALI for Epic.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.date: 06/01/2023
+---
+
+# Frequently asked questions about Epic on Azure Large Instances
+
+This article provides answers to frequently asked questions about Azure Large Instances (ALI) for Epic workloads.
+
+## In which regions is this service available?
+
+ALI on Epic is available in the following regions:
+
+* East US
+* US West2
+* US South Central  
+
+## Do I need to give permissions to allow the deployment of a managed resource group in my subscription?
+
+No, explicit permissions aren't required but you should register the resource provider with your subscription.
+
+## Why am I not able to see the ALI resources in Azure portal?
+
+Check Azure Policy set up if ALI managed RGs are not reflected in the portal.
+Azure subscription you use for Azure Large Instancesdeployments is already registered with the ALI infrastructure resource provider by the Microsoft Operations team during the provisioning process.
+If you don't see your deployed ALI instances under your subscription, register the resource provider with your subscription. 
+Ensure that your VNET address space provided in the request is the same as what you configure [Working with ALI in the Azure portal](../../work-with-ali-in-the-azure-portal.md)
+
+## Is it possible to have Azure ARC installed on Azure Large Instances?
+
+It’s not mandatory, but it is possible.
+If you need guidance, create a support ticket with Azure Customer Support so that Azure ARC Support can help in your set up.
```

Comment:
Shouldn't be "help with your setup"?

Reply:
Agree. Content Team please replace line 37 as: If you need Azure ARC Support, create a support ticket with Azure Customer Support.

Reply:
@suzizuber : You can use the link to ### Open a support request for ALI instances section under articles/azure-large-instances/work-with-ali-in-the-azure-portal.md article.
CC: @imsunnymaghosh